src/Arbitrable.ts

```diff
@@ -0,0 +1,22 @@
+import { Evidence } from '../types/schema'
+import { EthereumEvent, Bytes, BigInt, Address } from '@graphprotocol/graph-ts'
+import { EvidenceSubmitted as EvidenceSubmittedWithoutArbitrator, EvidenceSubmitted1 as EvidenceSubmittedWithArbitrator } from '../types/templates/Arbitrable/Arbitrable'
+
+export function handleEvidenceSubmittedWithoutArbitrator(event: EvidenceSubmittedWithoutArbitrator): void {
```

Comment:
Perhaps we can leave a comment here to explain why there's two different versions.

As we migrate more and more over time, I was also wondering if we should document some of these "migrations" somewhere, to keep track of all the different historical contract changes we've had to make or support with the subgraph.